pulse2percept/tests/test_electrode2currentmap.py

```diff
@@ -13,6 +13,44 @@ def test_TimeSeries():
                      data_orig.shape[-1]/resample_factor)
 
 
+def test_get_pulse():
+    for pulse_type in ['cathodicfirst', 'anodicfirst']:
+        for pulse_dur in [0.25/1000, 0.45/1000, 0.65/1000]:
+            for interphase_dur in [0, 0.25/1000, 0.45/1000, 0.65/1000]:
+                for tsample in [5e-6, 1e-5, 5e-5]:
+                    # generate pulse
+                    pulse = e2cm.get_pulse(pulse_dur, tsample,
+                                           interphase_dur,
+                                           pulse_type)
+
+                    # predicted length
+                    pulse_gap_dur = 2 * pulse_dur + interphase_dur
+
+                    # make sure length is correct
+                    npt.assert_equal(pulse.shape[-1],
+                                     int(np.round(pulse_gap_dur /
+                                                  tsample)))
+
+                    # make sure amplitude is correct: negative peak,
+                    # zero (in case of nonnegative interphase dur),
+                    # positive peak
+                    if interphase_dur > 0:
+                        npt.assert_equal(np.unique(pulse),
+                                         np.array([-1, 0, 1]))
+                    else:
+                        npt.assert_equal(np.unique(pulse),
+                                         np.array([-1, 1]))
+
+                    # make sure pulse order is correct
+                    idxMin = np.where(pulse == pulse.min())
```

Comment:
Sorry -- another PEP8 nit: please don't use camelCase. For example, here: `idx_min` would be good.

Excellent that you're adding tests. The more we add now, the happier we will be in the long run!